.github/workflows/main.yml

```diff
@@ -4,11 +4,11 @@ on: [push]
 
 jobs:
   test:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
     strategy:
       matrix:
         python-version: [3.9, 3.8]
-        poetry-version: [1.1.7]
+        poetry-version: [1.3.1]
```

Comment:
lock file was incompatible with old version, someone prob pushed a newer one, bumped this